crates/bitwarden-wasm-internal/src/pure_crypto.rs

```diff
@@ -159,6 +160,11 @@ impl PureCrypto {
         Ok(key.to_encoded())
     }
 
+    /// Wraps (encrypts) a spki der encoded encapsulation (public) key using a symmetric wrapping
```

Comment:
nit: unless SPKI is pronounced as an acronym (i.e. `spiki`) this should be `an` (which I'm guessing we aren't?)
```suggestion
    /// Wraps (encrypts) an spki der encoded encapsulation (public) key using a symmetric wrapping
```